liquibase-couchbase/src/test/java/system/change/InsertDocumentsFromFileSystemTest.java

```diff
@@ -36,9 +37,11 @@
 import static org.junit.jupiter.api.Assertions.assertEquals;
 
 
-public class InsertFromFileSystemTest extends LiquibaseSystemTest {
+public class InsertDocumentsFromFileSystemTest extends LiquibaseSystemTest {
     private static final Collection collection = bucketOperator.getCollection(TEST_COLLECTION, TEST_SCOPE);
     private static final int VALID_DOCS_COUNT = 2;
+    private static final String EXTRA_FIELD = "extraField";
+    private static final Keyspace keyspace = Keyspace.keyspace(TEST_BUCKET, TEST_SCOPE, TEST_COLLECTION);
```

Comment:
I think you can remove this definition and use `TestConstants.TEST_KEYSPACE` instead(same in InsertDocumentsSystemTest class and in few places as well)

Reply:
Done